connexion/apis/abstract.py

```diff
@@ -6,35 +6,25 @@
 import pathlib
 import sys
 import typing as t
-from enum import Enum
 
-from starlette.requests import Request
-
-from connexion.datastructures import NoContent
-from connexion.decorators.parameter import parameter_to_arg
 from connexion.exceptions import ResolverError
 from connexion.http_facts import METHODS
 from connexion.jsonifier import Jsonifier
-from connexion.lifecycle import ConnexionResponse
 from connexion.operations import make_operation
 from connexion.options import ConnexionOptions
 from connexion.resolver import Resolver
 from connexion.spec import Specification
-from connexion.utils import is_json_mimetype
 
 MODULE_PATH = pathlib.Path(__file__).absolute().parent.parent
 SWAGGER_UI_URL = "ui"
 
 logger = logging.getLogger("connexion.apis.abstract")
 
 
-class AbstractAPIMeta(abc.ABCMeta):
```

Comment:
Don't see a reason for this to be a Metaclass. I would still like to rework how the jsonifier is defined and passed around.